apps/labs/posts/alt-text-scipy-2022.md

```diff
@@ -0,0 +1,77 @@
+---
+title: SciPy 2022 Alt Text Scavenger Hunt
+author: Tony Fast
+published: July 12, 2022
+description: 'Lorem ipsum dolor sit amet'
+category: [Accessbility, Open SOurce]
+# featuredImage:
```

Comment:
You can leave featured image out, but the page is not able to render correctly without a hero image.